src/library/LocalLoopSaturationDirectActingMin.py

```diff
@@ -0,0 +1,63 @@
+"""
+## Local Loop Performance Verification - Direct Acting Loop Actuator Minimum Saturation
+
+### Description
+
+This verification checks that a direct acting control loop would saturate its actuator to minimum when the error is consistently below the set point.
+
+### Verification logic
+
+If the sensed data values are consistently below its set point, and after a default of 1 hour, the control command is still not saturated to minimum, then the verification fails; Otherwise, it passes.
+
+### Data requirements
+
+- feedback_sensor: feedback sensor reading of the subject to be controlled towards a set point
+- set_point: set point value
+- cmd : control command
+- cmd_min: control command range minimum value
+
+"""
+
+import pandas as pd
+from checklib import RuleCheckBase
+
+
+class LocalLoopSaturationDirectActingMin(RuleCheckBase):
```

Comment:
Would it make sense to have only one verification that would handle min/max? The logic is essentially the same.

Reply:
similar to above, I think min and max are for sensor reading on two sides of the set points separately.